spec/models/user_spec.rb

```diff
@@ -46,5 +59,62 @@
         end
       end
     end
+
+    describe "#tags" do
+      let(:user) { create(:user, :with_followed_artists, followed_artist_count: 1) }
+
+      it "lists all the tags created by the user" do
+        expect(user.tags).not_to be_empty
+        expect(user.tags.count).to eq(2)
+      end
+    end
+
+    describe "#delete_tags" do
+      let(:user) { create(:user, :with_followed_artists, followed_artist_count: 2) }
+
+      context "when deleting one tag" do
+        context "when user has tags" do
+          it "removes the tag" do
+            tags_count_before_delete = user.tags.count
+            user.delete_tags(user.tags.first.name)
+            expect(user.tags.count).to eq(tags_count_before_delete - 1)
```

Comment:
`expect { user.delete_tags(user.tags.first.name) }.to change(user.tags.count).by(-1)`

Reply:
Bien vu ✅